README.adoc

```diff
@@ -480,7 +486,11 @@ This is the same as "the outcome with higher odds".
 |mostTrusted
 |Choose the outcome that's backed by other users with the highest average return-on-investment. *Requires at least some data gathered beforehand via the 'recordChatsPredictions' setting!* E.g. only users with at least 5 made predictions are taken into account.
 |Take the return-on-investment from people who already placed a prediction, calculate the average per outcome, then pick the highest.
-|
+|* minTotalBetsPlacedByUser:
+
+* minTotalBetsPlacedOnPrediction:
+
+* minTotalBetsPlacedOnOutcome:
```

Comment:
Will need to add a description on what these params changes

Reply:
Have pushed the description.